content/pt/docs/referencia-da-api/pix/relato-infracao/fechar-relato-infracao/_index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Fechar Relato de Infração"
+linkTitle: "Fechar Relato de Infração"
+date: 2022-03-08T15:17:00-03:00
+lastmod: 2022-03-08T15:20:00-03:00
+weight: 9
+description: >
+  
+---
+
+Através desse endpoint será possível fechar um relato de infração.
```

Comment:
```suggestion
Através desse endpoint será possível que um participante indireto passa fechar um relato de infração.
```
we need to warn that this endpoint is for the exclusive use of indirect participants :)